apps/web/screens/Organization/PublishedMaterial/components/PublishedMaterialItem/PublishedMaterialItem.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Box, FocusableBox, Text, Tag } from '@island.is/island-ui/core'
+import { EnhancedAsset } from '@island.is/web/graphql/schema'
+import { useDateUtils } from '@island.is/web/i18n/useDateUtils'
+
+const getFileEnding = (url: string): string => {
+  const lastChunk = url.split('/').pop()
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
This eslint directive doesn't look necessary. Is this acting weird?

Reply:
This is something that I forgot, that function used to be different and I changed it at some point.